app/models/option.rb

```diff
@@ -5,6 +5,9 @@ class Option < ActiveRecord::Base
   validates :title, presence: true
   validates :description, presence: true
 
+  after_update  :draft!
```

Comment:
I'd suggest `draft_poll` or `draft_parent` to make it explicit that it's not an Option getting drafted, but a Poll as a whole